server/src/main/java/org/eclipse/lsp/cobol/core/model/variables/ElementItem.java

```diff
@@ -51,25 +54,20 @@ public ElementItem(
   @Override
   public Variable rename(RenameItem newParent) {
     return new ElementItem(
-        levelNumber,
-        name,
-        definition,
-        newParent.global,
-        newParent,
-        picClause,
-        value,
-        usageFormat);
+        levelNumber, name, definition, newParent.global, newParent, picClause, value, usageFormat);
+  }
+
+  @Override
+  public StructureType getStructureType() {
+    return ELEMENT_ITEM;
   }
 
   @Override
   public String getFormattedDisplayLine() {
     StringBuilder stringBuilder = new StringBuilder(getFormattedSuffix());
-    if (picClause != null)
-      stringBuilder.append(" PIC ").append(picClause);
-    if (usageFormat != UsageFormat.UNDEFINED)
-      stringBuilder.append(" USAGE ").append(usageFormat);
-    if (StringUtils.isNoneBlank(value))
-      stringBuilder.append(" VALUE ").append(value);
+    if (picClause != null) stringBuilder.append(" PIC ").append(picClause);
```

Comment:
Better to extract `PIC, VALUE`,  etc. ... as CONST, e.g.  repeatedly used in `ConditionDataName`, `IndependentDataItem` , `MultiTableDataName`, `TableDataName`..   then only remains add padding either with Java or using Guava methods

Reply:
I will do it as a separate PR, maybe altogether with setting DISPLAY as a default usage.